### PR TITLE
arash/mt5 dashboard

### PR DESCRIPTION
**Pull request by arash-binary**

### CODE REVIEW

File: src/templates/app/user/metatrader.jsx [hunk 1]
```diff
@@ -180,6 +180,18 @@ const Metatrader = () => (
...
                             </div>
                         </div>
+                        <div className='mt-sidebar-button'>
+                            <div className='small-icon'>
+                                <a href={it.url_for('metatrader/download')}>
+                                    <img src={it.url_for('images/pages/metatrader/dashboard/img-app-mac@2x.png')} />
+                                </a>
+                            </div>
+                            <div className='mt-link'>
+                                <a href='https://trade.mql5.com'>
```
Comment (ashkanx) [inline src/templates/app/user/metatrader.jsx]: @arash-binary  please add `target` and `rel` attributes

Comment (arash-binary) [reply src/templates/app/user/metatrader.jsx]: Sure

Comment (arash-binary) [reply src/templates/app/user/metatrader.jsx]: @ashkanx ` target='_blank' rel='noopener noreferrer'` is it right?

Comment (ashkanx) [reply src/templates/app/user/metatrader.jsx]: yes

**Pull request merged**